mocks/mock-bip-claims-api/src/main/java/gov/va/vro/mockbipclaims/model/bip/response/SpecialIssueTypesResponse.java

```diff
@@ -0,0 +1,13 @@
+package gov.va.vro.mockbipclaims.model.bip.response;
+
+import gov.va.vro.mockbipclaims.model.bip.ProviderResponse;
+import gov.va.vro.mockbipclaims.model.bip.SpecialIssueType;
+import lombok.Data;
+
+import java.util.List;
+
+/** SpecialIssueTypesResponse. */
+@Data
+public class SpecialIssueTypesResponse extends ProviderResponse {
+  private SpecialIssueType[] codeNamePairs;
```

Comment:
@dfitchett Question about the removing commented code comment.  Removing this specific comment introduces a new compiler warning since all public members require a javadoc comment.  I am interested in your thoughts about that case.  I would personally prefer to keep the comment and reduce our compiler warnings.  In generally though, i agree that useless comments are better to remove.